scripts/tvm_cli/build.sh

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env bash
+# Copyright 2020 Autoware Foundation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+IMAGE_NAME="autoware/model-zoo-tvm-cli"
+TAG_NAME="local"
+FROM_ARG="ubuntu:18.04"
+
+function usage() {
+    echo "Usage: $0 [OPTIONS]"
+    echo "    -c,--cuda              Build TVM cli with cuda enabled."
+    echo "    -h,--help              Display the usage and exit."
+    echo "    -i,--image-name <name> Set docker images name."
+    echo "                           Default: $IMAGE_NAME"
+    echo "    -t,--tag <tag>         Tag use for the docker images."
+    echo "                           Default: $TAG_NAME"
+    echo ""
+}
+
+OPTS=$(getopt --options chi:t: \
+         --long cuda,help,image-name:,tag: \
+         --name "$0" -- "$@")
+eval set -- "$OPTS"
+
+while true; do
+  case $1 in
+    -c|--cuda)
+      FROM_ARG="nvidia/cuda:10.1-devel-ubuntu18.04"
```

Comment:
I think arm64 builds will need their own cuda image here: "nvidia/cuda-arm64:11.1-devel-ubuntu18.04".

Reply:
@ambroise-arm that is a good point. we will add aarch64 + cuda support in a separate patch.